tests/e2e/taqueria-cli-commands.spec.ts

```diff
@@ -514,6 +577,50 @@ describe("E2E Testing for taqueria CLI,", () => {
         }
     })
 
+    test('Verify that ligo and archetype expose the plugin choice option for compile in the help menu', async () => {
+        try {
+            await exec(`taq install ../../../taqueria-plugin-ligo -p ${taqueriaProjectPath}`)
+            await exec(`taq install ../../../taqueria-plugin-archetype -p ${taqueriaProjectPath}`)
+            
+            // TODO: This can removed after this is resolved:
+            // https://github.com/ecadlabs/taqueria/issues/528
+            try {
+                await exec(`taq -p ${taqueriaProjectPath}`)
+            }
+            catch (_) {}
+
+            const ligoHelpContents = await exec(`taq --help --projectDir=${taqueriaProjectPath}`)
+            expect(ligoHelpContents.stdout).toBe(contents.helpContentsLigoArchetype)
+
+            await exec(`taq uninstall @taqueria/plugin-ligo -p ${taqueriaProjectPath}`)
+            await exec(`taq uninstall @taqueria/plugin-archetype -p ${taqueriaProjectPath}`)
+        } catch(error) {
+            throw new Error (`error: ${error}`)
+        }
+    })
+
+    test('Verify that ligo and archetype expose the plugin choice option for compile in the help menu', async () => {
+        try {
+            await exec(`taq install ../../../taqueria-plugin-ligo -p ${taqueriaProjectPath}`)
+            await exec(`taq install ../../../taqueria-plugin-archetype -p ${taqueriaProjectPath}`)
+
+            // TODO: This can removed after this is resolved:
+            // https://github.com/ecadlabs/taqueria/issues/528
+            try {
+                await exec(`taq -p ${taqueriaProjectPath}`)
+            }
+            catch (_) {}
```

Comment:
Do we want to add more descriptions if an exception occurs?

Reply:
Since this is a workaround and the task for it was assigned to Jimmy a few days ago I think we can get by with not putting much in here. If things go wrong then the test should fail and we can diagnose if needed